src/App.css

```diff
@@ -1,38 +1,134 @@
-.App {
-  text-align: center;
+*{
+  padding: 0;
+  box-sizing: border-box;
+  font-family: "Gill Sans", "Gill Sans MT", Calibri, "Trebuchet MS", sans-serif;
+  color: red;
+
+}
+body{
+  background-color: lightblue;
+  color: black;
+  padding: 40px;
+  font-family: Arial;
+  
+  margin-left: 30%;
+  margin-top: 5%
 }
+ 
 
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
+button:hover{
+  background-color: black;
+}
+#Start{
+  margin-right:1200px;
+  width: 100px;
+  height: 50px;
+  font-size:45px;
+  
 }
 
-@media (prefers-reduced-motion: no-preference) {
-  .App-logo {
-    animation: App-logo-spin infinite 20s linear;
-  }
+.north{
+  grid-area: 1/2/2/3;
+  /* width: 50px;
+  height: 50px; */
+  /* display: flex;
+  justify-content: space-around; */
+  font-size:25px;
+  /* Margin-left: 700px;
+  Margin-top: -10% */
```

Comment:
sections like this can cause a little bit of clutter through your code and make it harder to understand what's happening in your CSS. If this is CSS that you plan on using in the future you can leave a note about it, or organize each section so that the commented out chunk is separate than what's being used rather than having them mixed together